src/lib/resource/interceptor/token-refresher.es

```diff
@@ -54,8 +43,8 @@ class TokenRefresher {
   **/
   constructor (opts) {
     this.resetAttempts()
-    if (tokenRefresherSingletonInstance) {
-      return tokenRefresherSingletonInstance
+    if (this.tokenRefresherSingletonInstance) {
```

Comment:
Then create a function clear() or reset() that deletes the instance. It's a better solution IMO